fix: build error

HeIIow2 opened the pull request:
Hi,

Your build error was just a small configuration error. Nothing big.   
I'd guess the build tool had an update or something ifk. xD

But now it works. I hope I could help,   
Cheers <333

HeIIow2 committed:
fix: fixed build error

HassaanAkbar commented:
I tried upgrading build tool versions, they were quite beta level when I created the project. Build succeeds now. But the generated package has some issue. Can you try this,

```
npm run build
npm run pack
```

Then try that tgz path in docs and see if it gets built,
https://github.com/HassaanAkbar/vue-zoomable/blob/main/docs/package.json#L25
```
  "dependencies": {
    "vue-zoomable": "../vue-zoomable-1.1.2.tgz"
  }
----------------
cd docs
pnpm i
pnpm run build
```

HassaanAkbar commented:
I just noticed your name is not showing up in contributors. You might have some issue in commit info.

HeIIow2 commented:
> I just noticed your name is not showing up in contributors. You might have some issue in commit info.

I'ma look into it if I can solve it, else fuck it, I got money for this xD



> I tried upgrading build tool versions, they were quite beta level when I created the project. Build succeeds now. But the generated package has some issue. Can you try this,
> 
> ```
> npm run build
> npm run pack
> ```
> 
> Then try that tgz path in docs and see if it gets built, https://github.com/HassaanAkbar/vue-zoomable/blob/main/docs/package.json#L25
> 
> ```
>   "dependencies": {
>     "vue-zoomable": "../vue-zoomable-1.1.2.tgz"
>   }
> ----------------
> cd docs
> pnpm i
> pnpm run build
> ```

Ok so we deployed today, thus I had to [upload it to npm myself](https://www.npmjs.com/package/vue-zoomable-cute).   [Here](https://github.com/HeIIow2/vue-zoomable/tree/fork/selfMaintained) is my working branch, you could look at what changed. 

Here the changes required to build it:

1. you need to add an export for the css in `package.json`

```json
    "./dist/style.css": {
      "import": "./dist/style.css",
      "require": "./dist/style.css"
    }
```

2. The stuff I changed in this pr.
3. Remove either one of the export statements. I removed the non default one, but the other one should work as well. Beware that this could very easily break compatibility, so be careful there. The file I am talking about is: `src/index.ts`
4. You need to change the one already existing export in `package.json` to:

```json
    ".": {
      "import": "./dist/vue-zoomable.es.js",
      "require": "./dist/vue-zoomable.umd.js"
    },
```

Also note, you might wanna add your license to the `package.json` because npm doesn't know you use npm.

Cheers and good luck! <33
